clang/lib/StaticAnalyzer/Checkers/WebKit/UncountedLocalVarsChecker.cpp

```diff
@@ -190,11 +190,7 @@ class UncountedLocalVarsChecker
     if (shouldSkipVarDecl(V))
       return;
 
-    const auto *ArgType = V->getType().getTypePtr();
-    if (!ArgType)
```

Comment:
Some of these null checks may still be necessary (with `QualType.isNull()`).

Reply:
I was able to build the entire WebKit with this PR applied so it seems that we don't need these isNull checks in practice.

Reply:
Fair enough!